starter-code/src/functions-and-arrays.js

```diff
@@ -1,100 +1,224 @@
 // Find the maximum
+function maxOfTwoNumbers(first, second){
+  if (first >= second) 
+  {
+  return first;
+  } 
+  else 
+   {
+    return second;
+  } 
+}
+
+var largest = maxOfTwoNumbers(4, 6);
+console.log(largest);
 
 // Finding Longest Word
 var words = [
-  'mystery',
-  'brother',
-  'aviator',
-  'crocodile',
-  'pearl',
-  'orchard',
-  'crackpot'
+  "mystery",
+  "brother",
+  "aviator",
+  "crocodile",
+  "pearl",
+  "orchard",
+  "crackpot"
 ];
 
-// Calculating a Sum
+function findLongestWord(words){
+  
+  let longestWord = '';
+  
+  for(i=0;i < words.length; i++){
+    if ( words[i].length > longestWord.length ) {
+      longestWord = words[i];
+    }
+  }
+  console.log(longestWord);
+  if(longestWord.length === 0){
```

Comment:
You did not need to make an if statement for "undefined"...
If you would have just just changed line 29 to `  let longestWord = words[0];` , then you could have done this:
```js
function findLongestWord(words) {

  let longestWord = words[0];

  for (i = 0; i < words.length; i++) {
    if (words[i].length > longestWord.length) {
      longestWord = words[i];
    }
  }

    return longestWord;

}
var longest = findLongestWord(words);
console.log(longest);
```